apps/mobile/src/utils/feature-flag.ts

```diff
@@ -1,3 +1,57 @@
+import { useEffect } from 'react';
+import { Platform } from 'react-native';
+
+import { AutoEnvAttributes, ReactNativeLDClient } from '@launchdarkly/react-native-client-sdk';
+import * as Application from 'expo-application';
+
+function getDeviceId() {
+  if (Platform.OS === 'android') {
+    return Promise.resolve(Application.getAndroidId());
+  } else {
+    return Application.getIosIdForVendorAsync();
+  }
+}
+
+// TODO: this is used for localization and it should be revised in favor
+// of the proper feature flagging with launchdarkly
 export function isFeatureEnabled() {
   return process.env.NODE_ENV === 'development';
 }
+
+export const featureFlagClient = new ReactNativeLDClient(
+  // TODO: do not fallback to empty string
+  process.env.EXPO_PUBLIC_LAUNCH_DARKLY ?? '',
+  AutoEnvAttributes.Enabled,
+  {
+    debug: process.env.NODE_ENV === 'development',
+    applicationInfo: {
+      id: 'leather-mobile-wallet',
+      version: Application.nativeApplicationVersion ?? '0.0.1',
+    },
+  }
+);
+
+export const featureFlagKeys = {
+  notifications: 'notifications',
+};
+
+export type FEATURE_FLAG_KEYS = keyof typeof featureFlagKeys;
+
+async function setupFeatureFlags() {
+  const deviceId = await getDeviceId();
+  if (!deviceId) {
+    // TODO: handle this error properly
+    throw new Error('No device id detected');
+  }
+  featureFlagClient.identify({ kind: 'user', key: deviceId }).catch((e: any) => {
+    // TODO: analytics?
```

Comment:
Analytics is setup so we can add an event here

Reply:
Shit I didn't realize we don't have sentry.